consensus/consensus-types/src/proof_of_store.rs

```diff
@@ -132,6 +132,10 @@ impl BatchInfo {
     pub fn gas_bucket_start(&self) -> u64 {
         self.gas_bucket_start
     }
+
+    pub fn is_expired(&self) -> bool {
+        self.expiration() < aptos_infallible::duration_since_epoch().as_micros() as u64
```

Comment:
This doesn't seem right, should be ` self.expiration() > aptos_infallible::duration_since_epoch().as_micros() as u64`?

Reply:
A batch is expired if the current time is higher than self.expiration() right?

Reply:
Duh, I am dumb  😅